tools/metadata_merger/test-data/metadata-test.tsv

```diff
@@ -0,0 +1,6 @@
+sample_name	class   sampleType  injectionOrder  batch
```

Comment:
this is not a valid TSV file
![Screenshot_3_1_21__1_05_PM](https://user-images.githubusercontent.com/1814954/109494956-e2908980-7a8e-11eb-961e-6c3288374764.png)

Reply:
I noticed that there was something wrong with that but thought it's just a problem with interpreter on my machine. @hechth, that's the one that was in the Galaxy history you shared with me. Maybe there is some problem with the way we generate tables with metadata? Or it has just corrupted for some reason...